### PR TITLE
V mapetr/additional fixes in generation

### PR DESCRIPTION
**Pull request by vmapetr**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,17 +15,32 @@ Contributions to this project are [released](https://help.github.com/articles/gi
...
+|   └──templates/
+├── builders/
+├── helpers/
+├── installers/
+└── tests/
+    └──sources/
+```
+- `azure-pipelines` - contains global YAML definitions for build pipelines. Reusable templates for specific jobs are located in `templates` subfolder.
+- `builders` - contains Python builder classes and functions.
+- `helpers` - contains global helper funcations and functions.
```
Comment (MaksimZhukov) [inline CONTRIBUTING.md:41]: Typo

File: builders/nix-python-builder.psm1 [hunk 2]
```diff
@@ -41,8 +41,8 @@ class NixPythonBuilder : PythonBuilder {
         $this.Platform = $platform.Split("-")[0]
         $this.PlatformVersion = $platform.Split("-")[1]
 
-        $this.InstallationTemplateName = "nix-setup-template.sh"
-        $this.InstallationScriptName = "setup.sh"
+        $this.InstallationTemplateName = "nix-setup-template.ps1"
+        $this.InstallationScriptName = "setup.ps1"
```
Comment (maxim-lobanov) [inline builders/nix-python-builder.psm1:45]: Can we move `InstallationScriptName` to base class?

Comment (vmapetr) [reply builders/nix-python-builder.psm1]: Fixed

File: CONTRIBUTING.md (hunk 1 above)
Comment (vmapetr) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**